wrapper.go

```diff
@@ -69,13 +69,15 @@ func (s *LogFormatter) Format(entry *log.Entry) ([]byte, error) {
 // WrapHandler wraps the lambda handler
 func WrapHandler(handler interface{}, conf *Config) interface{} {
 	if err := loadConfig(*conf); err != nil {
+		defer recoverAndCheckFailWriteSpan()
```

Comment:
maybe I don't understand "defer" correctly, but it means that `recoverAndCheckFailWriteSpan` will always run after this block will finish, right?
But if this is the case, then there shouldn't be any file with `_end` in `SPANS_DIR`, and then we will create the file `balagan_end`, right?
But we shouldn't create this file now. Because everything is works as expected 🤔

Reply:
Here it doesn't work as expected, we're in `err`.

Why do we use `defer` here? Isn't it simpler to call the function?